cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -1,39 +1,121 @@
 # -*- coding: utf-8 -*-
+import fauxfactory
 import pytest
 
-timelines_reports = ('hosts', 'vm_operation', 'policy_events', 'policy_events2')
+from cfme.control.explorer import policies
+from cfme.tests.cloud_infra_common.test_events import vm_crud # noqa
+from cfme.infrastructure.provider import InfraProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
 
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers'),
+    pytest.mark.tier(2),
+    pytest.mark.provider([InfraProvider], required_fields=['provisioning'], scope='module')
```

Comment:
Required:
Please, we are not using `module` scope for provider setup so better to move it to `fixture`

Reply:
@digitronik, will moving the scope to `function` try to setup the provider for each test that my function is parametrized against?

Reply:
Yup, It will try to `setup` provider for each call. but if already setup then it will just `pass`